specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2022-05-01-preview/Operations.json

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
We discussed this yesterday. Updating this from -privatepreview to -preview would mean that SDK would get generated with preview version and this won't work as you do not have API support for 2022-05-01-preview version yet. You would need to make corresponding change in the API version.cs. Wait for M-train deployment to complete. Complete the ARM manifest deployment to include 2022-05-01-preview version. Only then, SDK with 2022-05-01-preview version will work.

Reply:
Yes, Just got this approved and will follow the process that you suggested and migrations CLI api will get delayed for now